src/main/java/nl/hsac/fitnesse/symbols/NextXDay.java

```diff
@@ -0,0 +1,58 @@
+package nl.hsac.fitnesse.symbols;
+
+import fitnesse.util.Clock;
+import fitnesse.wikitext.parser.Symbol;
+import fitnesse.wikitext.parser.Today;
+import fitnesse.wikitext.parser.Translator;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+/**
+ * Abstract class to get the next occurrence of a specific weekday. Can be called with +x to skip x occurrences
+ */
+
+public abstract class NextXDay extends Today {
+    private int DAY;
```

Comment:
Java standard for (non-static) fields is camel case (e.g. `day`). Is there a reason you chose capitals here?